src/main/java/org/folio/rest/impl/PostGobiOrdersHelper.java

```diff
@@ -223,7 +255,71 @@ public PostGobiOrdersHelper(HttpClientInterface httpClient,
     }
   }
 
-  public CompletableFuture<Map<Field, DataSource>> lookupOrderMappings() {
+  public CompletableFuture<String> lookupWorkflowStatusId(String workflowStatusCode) {
```

Comment:
Are there unit tests for the these new lookups?

Reply:
No... That's a good point. I don't see any unit tests for any of these lookup translations except for the order mappings lookup. I'll add some.